website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -0,0 +1,172 @@
+
+## Protect sensitive information using Unleash
+
+One of the biggest concerns for enterprise teams is preventing the accidental exposure of Personally Identifiable Information (PII). Unleash has implemented multiple layers of protection to [minimize PII risks in feature flag metadata](/topics/feature-flags/feature-flag-best-practices#6-protect-pii-by-evaluating-flags-server-side).
+
+We implement comprehensive encryption for data both in transit and at rest, utilizing industry-standard protocols. This encryption framework satisfies:
+
+-   **FedRAMP**: Meets federal requirements by using FIPS 140-2 validated encryption modules.
+-   **SOC 2**: Ensures protection of sensitive data through encryption, supporting the Confidentiality trust criterion.
+-   **ISO 27001**: Addresses cryptography control objectives and information transfer security requirements.
+
+### Protect data with Unleash frontend API
+
+Unleash also implements the data security principle of least privilege for [data collection](/understanding-unleash/data-collection). There's an important consideration when using the frontend API:
+
+-   The Frontend API sends context data to the Unleash instance for real-time feature flag evaluation.
+-   This data is used for real-time feature flag evaluation
+-   Unleash processes this data to evaluate feature flags but does not permanently store it
+-   The context might include non-PII data such as:
+    -   Device types
+    -   Application versions
+    -   Generic user roles
+    -   Geographic regions
+    -   Feature flag states
+
+With our approach, organizations can maintain strict data protection standards even in complex feature flag configurations.
+
+We collect two types of data: technical usage metrics and client metrics.
+
+Technical usage data includes feature flag usage, projects count, users, and API client SDKs. All technical data is anonymized and excludes sensitive information.
+
+Client metrics data is collected when SDKs evaluate feature flags. This includes:
+
+-   Feature flag name
+-   Environment
+-   Yes/no flag state
+-   Timestamp
+-   SDK version
+-   Client app name
+-   Instance ID
+
+Client metrics enable statistical analysis of feature deployments and flag state evaluations. The data powers Unleash's real-time insights dashboard for monitoring feature releases. Some key privacy points are:
+
+-   No PII or sensitive data is collected
+-   Metrics are anonymous
+-   Data collection can be disabled
+-   On-premise installations have full control over data collection settings
+-   Only flag evaluation data is stored, not user contexts
+
+All data helps improve platform stability, performance monitoring, and feature flag optimization while maintaining strict privacy standards.
+
+Learn how the Unleash architecture works in our [system overview](/understanding-unleash/unleash-overview#system-overview).
+
+### Use Unleash Edge for data privacy
```

Comment:
This section is way too lightweight.  See comments above about the front end API.  When we talk about architectural components, we need to contextualize them.   Also, we need to explain what we mean by context and why you would not want that to leave your network perimeter and how using Edge instead of the front end API helps with this.